### ISSUE TITLE
Running execute when creating a new account throws an error

### ISSUE THREAD

**Issue by codemaster101** [hashgraph/hedera-sdk-js#383]
Hi, I was following the example given in the documentation and I ran into this error:
```
(node:24067) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'encode' of undefined
    at AccountCreateTransaction._buildTransactions (/home/<redacted>/Desktop/test/hedera-test/node_modules/@hashgraph/sdk/lib/transaction/Transaction.cjs:713:58)
    at AccountCreateTransaction._makeRequest (/home/<redacted>/Desktop/test/hedera-test/node_modules/@hashgraph/sdk/lib/transaction/Transaction.cjs:694:10)
    at AccountCreateTransaction.execute (/home/<redacted>/Desktop/test/hedera-test/node_modules/@hashgraph/sdk/lib/Executable.cjs:233:28)

```
The following is my code that I used to implement the program.
```
const hedera = require('@hashgraph/sdk');
const hdkey = require('hdkey');
const _sodium = require('libsodium-wrappers-sumo');

const seed = Buffer.from('6c7968455be48a3b61af24ec75bc832fe8f908399dbeb7025495beaccc7e2ea6f5c9887585515cc0be748dd0e1f1bc17ad455c018d64c053a1a9a9bde7a35be3', 'hex');
const hdPath = 'm/49\'/3030\'/0\'/0\'';

async function generate(){//client) {
        console.log('Generating the account');
        const client = hedera.Client.forTestnet();
        client.setOperator('0.0.644464',  '302e020100300506032b657004220420738615a7accdaf1de92678f61be8b6888be329a45ad2a73a2993a135889c9fbd');
        await _sodium.ready;
        const sodium = _sodium;
        const hdKey = hdkey.fromMasterSeed(seed);
        const { privateKey, publicKey } = hdKey.derive(hdPath);
        const ed25 = await sodium.crypto_sign_seed_keypair(privateKey);
        const pubKey = new hedera.PublicKey(ed25.publicKey);
        const acc = await new hedera.AccountCreateTransaction()
                .setKey(pubKey)
                .setInitialBalance(hedera.Hbar.fromTinybars(1000))
                .execute(client);
        console.log(acc);
}

async function main() {
        await generate(/*client*/);
}

main();
```

It seems that the proto object at Transaction.cjs while building the transactions does not have the SignedTransaction function/object.
```
this._transactions.push({
        signedTransactionBytes: _proto.SignedTransaction.encode(this._signedTransactions[i]).finish()
      });
```

Anyone know what the issue is? Is it something to do with the kind of details I am passing into it?

EDIT:
Version 2.0.19 for @hashgraph/sdk

**Comment by mehcode** [comment]
Please `npm upgrade` or remove/reinstall the SDK. There was an intermettitent problem where you likely do not have the latest version of `@hashgraph/proto`. Check your lock file and be sure you have v1.1.2.

**Comment by codemaster101** [comment]
The issue is now fixed with the release of `2.0.20` yesterday. Thanks @mehcode
